src/kakarot/precompiles/ec_recover.cairo

```diff
@@ -73,3 +83,14 @@ namespace PrecompileEcRecover {
         return (32, output, GAS_COST_EC_RECOVER, 0);
     }
 }
+
+namespace EcRecoverHelpers {
+    func ec_point_equal(point_0: EcPoint, point_1: EcPoint) -> (is_equal: felt) {
+        if (point_0.x.d0 == point_1.x.d0 and point_0.x.d1 == point_1.x.d1 and
+            point_0.x.d2 == point_1.x.d2 and point_0.y.d0 == point_1.y.d0 and
+            point_0.y.d1 == point_1.y.d1 and point_0.y.d2 == point_1.y.d2) {
```

Comment:
can you just update the order of the equality for better readability? ie having only `x.d` in first column and `y.` in the second

Reply:
```
point_0.x.d0 == point_1.x.d0 and point_0.y.d0 == point_1.y.d0 and
point_0.x.d1 == point_1.x.d1 and point_0.y.d1 == point_1.y.d1 and
point_0.x.d2 == point_1.x.d2 and point_0.y.d2 == point_1.y.d2
```

Reply:
fixed